src/routes/workshop-a11y-fundamentals.svelte

```diff
@@ -203,174 +550,348 @@
 </svelte:head>
 
 <header class="header">
-  <ToggleTheme klass="toggleTheme" />
-  <!-- <Nav on:calculated={() => true} horizonSpace="10000" /> -->
+  <a href={SITE_URL} class="logo u-link">
+    Sandrina
+  </a>
+  <ToggleThemePage />
 </header>
 
-<main class="wrapper">
-  <h1 class="f-title title">Web Accessibility Fundamentals</h1>
-  <p class="rw">
-    Remote Workshop
-    <span class="rw-d">•</span>
-    <span class="rw-t">3 x 3hours</span>
+
+<main class="wrapper blogpost">
+  <header class="t-hero">
+    <h1 class="t-heroTitle t-title">Web Accessibility Fundamentals</h1>
+    <p class="t-heroMantra">The web is awesome and everyone should be able to enjoy it.</p>
+    <span class="t-heroKicker">Online Workshop</span>
+    <div class="t-heroAbout">
+      <!-- <p>April 11-13 · {eventHour} {eventTZ}</p> -->
+      <p class="t-heroCreator">with <img src="/sandrinap.jpg" alt=""  class="t-heroCreatorPic" />
+        <a class="u-link t-heroCreatorName" rel="noreferrer" href={TWITTER_URL} on:click={() => trackClick('csstricks_journey')}>
+          Sandrina Pereira
+        </a>
+      </p>
+    </div>
+  </header>
+
+  <div class="cta-sticky">
+    <!-- <button class="u-link">Coming soon!</button> -->
+  </div>
+
+  <span class="t-separator"></span>
+
+  <p class="t-p">
+    Let’s be honest, as web creators, we are still leaving accessibility as an afterthought. Learning how to create web apps using modern UI libraries sounds way more exciting, <i>I understand you...</i>
+  </p>
+
+  <h2 class="t-hook">
+      A11Y doesn't need to be boring
+  </h2>
+  
+    
+  <p class="t-p">
+    Perhaphs you already know that accessibility matters  and you are aware that color contrast is a thing.
+  </p>
+
+  <p class="t-p">
+    But... <i>what else?</i>
   </p>
 
-  <p class="p">
-    Hi there, I'm
-    <a href={SITE_URL} rel="noreferrer" class="u-link">Sandrina Pereira</a>
-    and I believe that making the web accessible is our duty as web creators.
+  <p class="t-p">
+    In your solo journey into learning accessibility, you’ve might have visited the official 
+    <a href="https://www.w3.org/WAI/" class="u-link">W3C WAI</a>
+    website and took a look at the
+    <a href="https://www.w3.org/TR/WCAG21" class="u-link">
+    WCAG 2.1 guidelines.
+    </a>
+  </p>
+
+  <p class="t-p">
+    But then you felt overwhelmed, lost,... and left it aside.
+  </p>
+
+  <h2 class="t-hook">
+      Let me simplify it for you
+  </h2>
+
+  <p class="t-p">
+    Let me show you why accessibility is worthy of being part of your skillset foundations, and how it benefits way more people that you judge, including you and me.
```

Comment:
...and show you why accessibility is worthy of being part of your skillset foundations, and how it benefits way more people than you imagine, including you and me.

Accessibility does not limit your solutions, on contrary, it makes them more inclusive.

(personally eu acho que fica melhor não repetir o "let me" e fazer frases mais declarativas, tem mais power e soa mais confiante. mas não estava errado)